packages/qwik/src/core/v2/client/vnode.ts

```diff
@@ -394,6 +396,43 @@ export const vnode_ensureElementInflated = (vnode: VNode) => {
   }
 };
 
+export const vnode_getDOMChildVNodes = (
```

Comment:
this is copy pasted code `vnode_getDOMChildNodes `, but returns VNodes instead of elements

Reply:
Can we somehow merge the code so that it is not duplicate? I don't want to be fixing the same code twice.

Maybe have an extra argument which you can use if the VNode should be unwraped or not.